src/index.js

```diff
@@ -58,22 +58,22 @@ const Alpine = {
         const observer = new MutationObserver((mutations) => {
             if (this.pauseMutationObserver) return;
 
-            for (let i=0; i < mutations.length; i++){
-                if (mutations[i].addedNodes.length > 0) {
```

Comment:
forEach is waaaay faster than our old-lovely and imperative for-loop.

Proof ([benchmark](https://www.measurethat.net/Benchmarks/Show/3862/array-loops-for-vs-for-in-vs-for-of-vs-foreach-vs-map-f))
![image](https://user-images.githubusercontent.com/4066435/85788722-1401ca00-b736-11ea-9fc7-940b46cede37.png)

Reply:
As you can see `for-of` loop is faster than `forEach` in Chrome.
But in Firefox `for-of` even slower than `for-loop` (see screenshot below)

So I've used `forEach` cause of that.

![image](https://user-images.githubusercontent.com/4066435/85790649-3a753480-b739-11ea-9198-04c126fb8b7b.png)